apps/core/components/ItineraryCard/TripSectorHelpers.js

```diff
@@ -1,11 +1,9 @@
 // @flow
 
 import * as DateFNS from 'date-fns';
-
-export const timeSimpleFormat = 'H:mm';
-export const dateFormat = 'ddd D MMM';
+import { HOURS_MINUTES_FORMAT } from '@kiwicom/margarita-config';
 
 export const getFormattedDate = (
   time: ?string,
-  format: ?string = timeSimpleFormat,
-) => DateFNS.format(DateFNS.parse(time), format);
+  format: ?string = HOURS_MINUTES_FORMAT,
+) => time && format && DateFNS.format(DateFNS.parseISO(time), format);
```

Comment:
Do you need to check if the format exist? because you set default value?

Reply:
Because `parseISO` don't want null.

```
[flow] Cannot call `DateFNS.parseISO` with `time` bound to `argument` because null or undefined [1] is incompatible with string [2]. (TripSectorHelpers.js:9:38)
```

Reply:
what about something like this?
```
export const getFormattedDate = (time: ?string, format: ?string) =>
  time &&
  DateFNS.format(DateFNS.parseISO(time), format ?? HOURS_MINUTES_FORMAT);

```

Reply:
I'm getting your point now, it makes sense now :)

Done